package.json

```diff
@@ -21,13 +21,18 @@
     "lint": "eslint . --quiet --ext .json --ext .js",
     "test": "jest",
     "build": "rollup -c",
-    "prepare": "yarn build"
+    "install": "yarn build"
   },
   "engines": {
-    "node": ">=8.10.0"
+    "node": ">=10.10.0"
   },
   "dependencies": {
-    "lodash": "^4.17.10"
+    "lodash": "^4.17.10",
+    "rollup": "^2.0.5",
```

Comment:
Needed to move rollup to a dependency for the build since yarn will not pre-install dev dependencies for lifecycle hooks unlike npm.

Reply:
Is there any way to consume it as straight es6 modules? i.e. bypass rollup entirely